src/util/execute-lifecycle-script.js

```diff
@@ -173,20 +173,26 @@ export async function executeLifecycleScript(
     conf.windowsVerbatimArguments = true;
   }
 
-  const stdout = await child.spawn(sh, [shFlag, cmd], {cwd, env, stdio, ...conf}, data => {
-    if (spinner) {
-      const line = data
+  let updateProgress;
+  if (spinner) {
+    const ticker = spinner.tick.bind(spinner);
```

Comment:
That being said, I think you could just use an invariant inside the `updateProgress` callback to tell Flow that `spinner` *does* exist.